roles/installer/ospd/images/tasks/build.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Check if there is a guest image is present
+  stat:
+      path: "/home/{{ installer.user.name }}/guest_image.qcow2"
+  register: stat_result
+
+- name: Download guest image for build if needed
+  get_url:
+      dest: "/home/{{ installer.user.name }}/guest_image.qcow2"
+      url: "{{ installer.images.guest_image }}"
```

Comment:
Installler does not have these settings. All the images info should be moved to the branch.